capstone_project/src/main/webapp/scripts/maps_script.js

```diff
@@ -100,3 +100,86 @@ function createArticleEntry(text, schoolName) {
   node.appendChild(a);
   document.getElementById(schoolName).appendChild(node);
 }
+
+/** Creates a map centered at the geographic center of the US and
+ * adds it to the page. */
+function createMap() { // eslint-disable-line no-unused-vars
+  const map = new google.maps.Map(
+      document.getElementById('map'),
+      {center: {lat: 39.828502, lng: -98.579512}, zoom: 4});
+
+  // Create a marker for each school. (TODO: fetch this info from server)
+  createMarker(map, 33.639982, -117.844682, 'UCI');
+  createMarker(map, 47.655277, -122.303606, 'UW');
+  createMarker(map, 47.758821, -122.190725, 'UWB');
+}
+
+/** Creates and returns the content string for a marker.
+ * @param {string} name
+ * @return {string}
+ */
+function createContentString(name) {
+  const contentString = '<div id="content">'+
+      '<div id="siteNotice">'+
+      '</div>'+
+      `<h1 id="firstHeading" class="firstHeading">${name}</h1>`+
+      '<div id="bodyContent">'+
+      `<p>Here you will find all the latest updates on <b>${name}</b>:</p>` +
```

Comment:
@dchin2 I think this might be an outdated version of my code? My most recent version of maps_script.js creates contentString using template literals on these lines.